src/client/components/pages/RoomAdmin/CreateRoomModal.tsx

```diff
@@ -205,10 +205,10 @@ const CreateRoomModal: FunctionComponent<CreateRoomModalProps> = function ({
       };
       isValid = false;
     }
-    if (Number.isNaN(parseInt(form.capacity, 10))) {
+    if (!/^\d+$/.test(form.capacity)) {
       updatedFormErrors = {
         ...updatedFormErrors,
-        capacity: 'Capacity is required to submit this form, and it must be a number.',
+        capacity: 'Capacity is required to submit this form, and it must be a positive number.',
```

Comment:
Technically, the regex above will allow one or more zeroes (e.g. `0000`). I think that's OK in the sense that one zero is sort of an "unknown" placeholder, but we should make sure we're converting this value to an integer when it's saved into the database. In either case, I think it's fine to leave off "positive".

Reply:
Ah ok, that's a good point about the admins possibly wanting to use `0` as a placeholder. That makes sense - I updated the language in [c345f15](https://github.com/seas-computing/course-planner/pull/581/commits/c345f15902f6fb3fe68bce21766aa084e45a7520).